user_guide/intro_log_in_and_edit_profile.rst

```diff
@@ -0,0 +1,554 @@
+in.
+
+.. caution::
+
+    It is strictly recommended to change your password immediately after the first login. You can learn how to do it in 
+    the :ref:Changing Your Password section `<user-guide-getting-started-profile-password>` below.
+
+If you have forgotten the credentials, click the *"Forgot your password?"* link and follow the wizard to restore them.
+
+
+Log In Using Your Google Credentials
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the Google Single Sign-On capability has been enabled for your organization, you will see the *"login using google"* 
+link at the login screen. This means you can log in using your corporate Gmail account credentials. Usually this 
+capability is enabled for corporate email addresses only, so your personal mailbox credentials won’t do.
+
+
+Log In Using Other Corporate Credentials 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+OroCRM can be set up to accept your existing corporate login and password. This is possible if your organization 
+utilizes a common server to share them across multiple applications. This is referred to as LDAP integration. Please 
+check with your system administrator to see if this is the case.
+
+
+.. _user-guide-getting-started-my-user:
+
+Access Your User Page
+---------------------
+
+You can always find a link to your user page in the drop-down menu that resides below your username in the top right 
+corner. This is a fast way to access your user profile, calendar, mailbox, and task list. 
+
+.. image:: ./img/intro/user_menu.png
+
+The following subsection is devoted to the user page that is available with *"My User"* menu. You can find more 
+information about other menu items in the :ref:`User Menu <user-guide-getting-started-user-menu>` document.
+
+.. _user-guide-getting-started-my-user-overview:
+
+My User Page Overview
+^^^^^^^^^^^^^^^^^^^^^
+
+This page provides you the full details of your user profile.
+
+Its *"General"* section contains the basic profile details.
+
+.. image:: ./img/intro/user_view_general.png
+
+The set of information available in the section depends on the 
+:ref:`permissions <user-guide-user-management-permissions-basic>` defined for your user. The information may include:
+
+* Your name and system information such as status (Active), login count, date and time of the last login - in the header
+
+* Your username 
+
+* Your birthday
+
+* Information about your role and place in the company structure, such as your roles in the system and business units 
+  you have access to, your API key (if any)
+  
+* Any additional details of a user defined for your organization (in the example above this is the *"Title"* field).
+
+* A link to Outlook Add-in download (for enterprise users only)
+
+* Tags assigned to your user
+
+If there is an email saved or/and a call logged in OroCRM related to you, they will be displayed in the 
+*"Activities"* section. 
```

Comment:
Edit: If there is an email saved and/or a call logged in OroCRM that is related to you, they will be displayed in the 
 +*"Activities"* section.